lib/cli/cmd-init.js

```diff
@@ -49,6 +49,10 @@ function runCliInit(folderPath, options) {
     initArgs.push('--type', options.type);
   }
 
+  if (options.yes) {
```

Comment:
I don't like having to do this for each option we want to pass down to donejs-cli.

I looked into getting any option automatically passed down `options.options` is a list of commander [Option](http://tj.github.io/commander.js/#Option) objects. There is a private API https://github.com/tj/commander.js/blob/master/index.js#L76-L78 to get the object attribute name (the filed set at the root of the `options` object) so we could programatically check each and see if they are set, then push the the `Option.long` (the name with the `--`).

But I decided not to do this, b/c well, relying on private API didn't seem safe.